### PR TITLE
ci: implement e2e test for Sink

### PR DESCRIPTION
**Pull request by nanderstabel**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

Add sink e2e test to CI.

## Checklist

~~- [ ] I have written necessary rustdoc comments~~
~~- [ ] I have added necessary unit tests and integration tests~~
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

**Comment by tabVersion** [comment]
I briefed the pr and I think this can be merged to branch `ns-sink-meta` instead of main

File: src/meta/src/manager/id.rs [hunk 1]
```diff
@@ -211,6 +213,7 @@ where
             IdCategory::Database => &self.database,
             IdCategory::Schema => &self.schema,
             IdCategory::Table => &self.table,
+            IdCategory::Sink => &self.sink,
```
Comment (tabVersion) [inline src/meta/src/manager/id.rs:216]: why CI relate to meta

Comment (nanderstabel) [reply src/meta/src/manager/id.rs:216]: hmm this is still leftover code from ns-sink-meta before it got merged with main. I will correct it 👍

File: ci/docker-compose.yml [hunk 2]
```diff
@@ -26,3 +27,45 @@ services:
...
+      - ..:/risingwave
+    pid: host
+    userns_mode: host
+    network_mode: host
+    # networks:
+    #   nw:
+    #     ipv4_address: 172.24.2.1
+
+  mysql:
+    image: mysql/mysql-server:latest
```
Comment (skyzh) [inline ci/docker-compose.yml:46]: would be better if we could specify a version (e.g. `mysql/mysql-server:8.0`)

Comment (nanderstabel) [reply ci/docker-compose.yml:46]: good point, will fix 👍

**Pull request closed without merging**